crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -300,6 +301,18 @@ def save_and_restore
       end
     end
 
+    def post_cleanup_repos
+      [:pool, :updates].each do |repo|
+        ret, message = ProvisionerService.new(Rails.logger).disable_repository(
+          "suse-12.1",
```

Comment:
I would not hardcode this, I think the better way would be to deactivate everything what is not suse-12.2 or cloud 7 in this case.